beacon_node/beacon_chain/src/data_availability_checker.rs

```diff
@@ -508,11 +508,23 @@ impl<T: BeaconChainTypes> DataAvailabilityChecker<T> {
         })
     }
 
+    /// Total count of columns in custody
+    pub fn get_custody_column_count(&self) -> usize {
+        let custody_subnet_count = if import_all_data_columns {
+            spec.data_column_sidecar_subnet_count as usize
+        } else {
+            spec.custody_requirement as usize
+        };
+
+        custody_subnet_count.saturating_mul(spec.data_columns_per_subnet())
+    }
```

Comment:
`import_all_data_columns` isn't available here, but I've added this function in another PR, feel free to copy it over if you'd like to test this: 

https://github.com/sigp/lighthouse/pull/6255/files#diff-eeea69875a56959be32e581165eace1a25fd517afd98fe0576372db24e0fd9a5R128-R132

Reply:
done